node/src/main.rs

```diff
@@ -62,59 +54,10 @@ enum ConnectionType {
     WS,
 }
 
-fn main() {
-    use std::sync::Mutex;
-    use tokio::runtime;
-
-    // Create components for tokio context: multi-threaded runtime, executor
-    // context on the runtime, and Timer handle.
-    //
-    // Configure the runtime to shutdown after a panic.
-    let runtime: Arc<Mutex<Option<runtime::Runtime>>> = Arc::new(Mutex::new(None));
-    let handler_runtime = runtime.clone();
-    *runtime.lock().unwrap() = Some(
-        runtime::Builder::new()
-            .core_threads(*TOKIO_THREAD_COUNT)
```

Comment:
Do we not run out of handles now with the default configuration? If so, I wouldn't be surprised if a bug was fixed by this change, since 100 threads is not a crazy amount. Did you find something?

Reply:
The default configuration for the core thread pool is the number of CPU cores, and for the blocking pool it's 1000 I believe. We were using a large core pool to work around doing blocking work in the core pool, but now that we use `spawn_blocking` almost everywhere that is solved so I put us back on the default threadpool size for simplicity.

Running out of file descriptors because of the thread pool is something that happens on mac because it has default limits that are too low, that limit can be raised.